_posts/2018-05-25-katuma-en-xifres.md

```diff
@@ -12,49 +12,22 @@ Algunes dades rellevants que s'han vist aquests darrers quatre mesos i escaig s
 
 Tot i que el nombre d'organitzacions amb presència a la plataforma ha crescut quasi de forma lineal des del passat octubre fins a les 84 organitzacions, només 25 productors i 18 grups han estat presents en alguna comanda.
 
-<p>
-  <div class="embed-responsive embed-responsive-16by9 video">
-    <iframe
-        src="http://katuma-business-intelligence.herokuapp.com/public/question/76aa262d-cb3e-4884-9f5d-1066c9495da7"
-        frameborder="0"
-        width="800"
-        height="600"
-        allowtransparency
-    ></iframe>
-  </div>
-</p>
+![Enterprises by day](/assets/post_images/2018/sum_enterprises_by_day.png){:width="100%"}
 
 La majoria de les organitzacions que no són actives han obert els seus comptes per poder visualitzar la seva entitat al mapa de la plataforma, mentre d'altres senzillament han fet algunes proves per veure com funciona.
 
 És interessant constatar però que hi ha un nombre major de productores que de grups, al contrari de la percepció que tenim els que participem del moviment de grups i cooperatives de consum. Aquest fet coincideix amb la recerca duta a terme per en Ricard Espelt al seu treball [«Consum autogestionat en l’era de la Societat Xarxa. El paper de les TIC en el consum cooperatiu de productes agroalimentaris»](http://www.cooperatives.barcelona/).
 
-<p>
-  <div class="embed-responsive embed-responsive-16by9 video">
-    <iframe
-        src="http://katuma-business-intelligence.herokuapp.com/public/dashboard/aea6ae01-d8a8-45b2-8298-fd827d0fb41a"
-        frameborder="0"
-        width="800"
-        height="400"
-        allowtransparency
-    ></iframe>
-  </div>
-</p>
+![Absolute numbers](/assets/post_images/2018/absolute_numbers.png){:width="100%"}
+*Dades maig 2018*
```

Comment:
I'd make this smaller so that it doesn't the same hierarchy level as the text body. This is how it looks like now.

![screenshot from 2018-07-12 18-04-20](https://user-images.githubusercontent.com/762088/42645390-27473da4-85fe-11e8-80cb-4f9ab7bb4e4f.png)

I think a `<small>` here would do the trick.